packages/moda-sdk/src/profile/getName.ts

```diff
@@ -0,0 +1,38 @@
+import { Address } from "viem";
+import { AddressError, Config, ConfigError } from "../types";
+import { Result } from "../utils/types";
+import { verifyConfig } from "../utils/verifyConfig";
+import { isValidAddress } from "../utils/isValidAddress";
+import { ProfileABI } from "./abi/ProfileABI";
+import { ProfileError } from "./types";
+
+interface GetNameArgs {
+    config: Config;
+    profileAddress: Address;
+}
+
+export async function getName(
```

Comment:
Let's use the same naming as the contract for all functions. `name` vs `getName`